requirements.txt

```diff
@@ -7,4 +7,6 @@ PyYAML==5.1.2
 requests==2.22.0
 six==1.12.0
 urllib3==1.25.3
--e git+https://github.com/exonum/exonum-python-client@master#egg=exonum-python-client
+# Use an editable git dependency on exonum-python-client till it is released.
```

Comment:
Nit: Not sure I understand the point of "till it is released" part. If I understand correctly, `requirements.txt` is used as a lockfile here, i.e., it specifies the *exact* versions of dependencies. Since `exonum-python-client` and `exonum-launcher` are tightly related, it may well be the case that the revision spec will be used most of the time.

Reply:
Possibly, I can't say given my limited knowledge of these projects :)